swmm-toolkit/src/swmm/toolkit/solver_rename.i

```diff
@@ -20,11 +21,11 @@
 %rename(report)                                     swmm_report;
 %rename(get_mass_bal_err)                           swmm_getMassBalErr;
 %rename(close)                                      swmm_close;
-%rename(version)                                    swmm_getVersionInfo;
+%rename(version)                                    swmm_getVersion;
```

Comment:
I think it would be good to finalize these swig function names before a release is made. Are we ok with using these function names? @cbuahin @michaeltryby

Reply:
@jennwuu What I tried to do here is move some code around so that the changes made to swmm5.h and swmm5.c are minimal. The function swmm_getVersionInfo has been moved to toolkit.c. What is happening here is that I'm wrapping the swmm_getVersion function found in the original swmm5 API. I agree we should go through and agree on naming. Also not sure why we need so many methods for getting version information.